src/metabolic_ninja/models.py

```diff
@@ -44,6 +44,7 @@ class DesignJob(TimestampMixin, db.Model):
     model_id = db.Column(db.Integer, nullable=False)
     product_name = db.Column(db.String, nullable=False)
     max_predictions = db.Column(db.Integer, nullable=False)
+    aerobic = db.Column(db.Boolean, nullable=True)
```

Comment:
Add a comment here to explain that null should not be used for new entries, but allowed to be compatible with the older jobs created before the field existed.